tests/core/operations/test_transform.py

```diff
@@ -117,65 +118,88 @@ def _unpad_rasters(sdata: SpatialData) -> SpatialData:
     return SpatialData(images=new_images, labels=new_labels)
 
 
-# TODO: when the io for 3D images and 3D labels work, add those tests
-def test_transform_image_spatial_image(images: SpatialData):
-    sdata = SpatialData(images={k: v for k, v in images.images.items() if isinstance(v, SpatialImage)})
+def _postpone_transformation(
+    sdata: SpatialData, from_coordinate_system: str, to_coordinate_system: str, transformation: BaseTransformation
+):
+    for element in sdata._gen_elements_values():
+        d = get_transformation(element, get_all=True)
+        assert isinstance(d, dict)
+        assert len(d) == 1
+        t = d[from_coordinate_system]
+        sequence = Sequence([t, transformation])
+        set_transformation(element, sequence, to_coordinate_system)
 
-    VISUAL_DEBUG = False
-    if VISUAL_DEBUG:
-        im = scipy.misc.face()
-        im_element = Image2DModel.parse(im, dims=["y", "x", "c"])
-        del sdata.images["image2d"]
-        sdata.images["face"] = im_element
 
-    affine = _get_affine(small_translation=False)
-    padded = transform(
-        transform(sdata, affine, maintain_positioning=False), affine.inverse(), maintain_positioning=False
-    )
-    _unpad_rasters(padded)
-    # raise NotImplementedError("TODO: plot the images")
-    # raise NotImplementedError("TODO: compare the transformed images with the original ones")
+def _remove_coordinate_system(sdata: SpatialData, coordinate_system: str):
+    for element in sdata._gen_elements_values():
+        remove_transformation(element, coordinate_system)
```

Comment:
Same as above

Reply:
moved to utils function `remove_transformations_to_coordinate_system()`